specs/order_spec.rb

```diff
@@ -37,6 +37,7 @@
       expect(order.fulfillment_status).must_equal fulfillment_status
     end
 
+# Don't understand what this one means
```

Comment:
This test is looking through each item in `valid_statuses` as values for `fulfillment_status` in an `Order`. It makes an instance of `Order` for each of these items, and then tests that the instance's value for `order.fulfillment_status` works and is correct.